Modules/selectmodule.c

```diff
@@ -2444,12 +2444,17 @@ _select_exec(PyObject *m)
         return -1;
     }
 
+#define ADD_INT(val) \
+    if (PyModule_AddIntConstant((m), #val, (val)) < 0) { \
+        return -1; \
+    }
```

Comment:
I'd prefer the `do...while(0)` macro idiom, so it can be used as a statement:

```suggestion
#define ADD_INT(VAL) do {                                 \
    if (PyModule_AddIntConstant((m), #VAL, (VAL)) < 0) {  \
        return -1;                                        \
    }                                                     \
while (0)
```

(I also prefer to align the line continuation characters, and to use capital letters for the macro argument)